### PR TITLE
reset rtv votes when a mapvote is cancelled

### PR DESCRIPTION
**Pull request by plally**

### CODE REVIEW

File: lua/mapvote/server/modules/rtv.lua [hunk 1]
```diff
@@ -1,5 +1,5 @@
-MapVote.RTV = MapVote.RTV or {}
-local RTV = MapVote.RTV
+local RTV = MapVote.RTV or {}
```
Comment (plally) [inline lua/mapvote/server/modules/rtv.lua:1]: lua_ls works better if its like this for some reason

Comment (sarahsturgeon) [reply lua/mapvote/server/modules/rtv.lua:1]: Yeah that bugs me - ran into the same issue before

File: lua/mapvote/server/modules/rtv.lua [hunk 2]
```diff
@@ -82,6 +82,12 @@ function RTV.Start()
     MapVote.Start()
 end
 
+function RTV.ResetVotes()
+    for _, ply in pairs( player.GetAll() ) do
```
Comment (sarahsturgeon) [inline lua/mapvote/server/modules/rtv.lua]: 🤓 erm technically
```suggestion
    for _, ply in ipairs( player.GetAll() ) do
```

**Review by sarahsturgeon — APPROVED** [review]

File: lua/mapvote/server/modules/rtv.lua (hunk 2 above)
Comment (plally) [reply lua/mapvote/server/modules/rtv.lua]: I can also change this to GetHumans instead

**Pull request merged**